src/fetch/account.ts

```diff
@@ -1,10 +1,16 @@
 // Credit to https://github.com/OpenZeppelin/openzeppelin-subgraphs
-import { Address } from "@graphprotocol/graph-ts";
-
 import { Account } from "../../generated/schema";
 
-export function fetchAccount(address: Address): Account {
-  let account = new Account(address.toHex());
+/**
+ * Searches for and returns an Account, initializing a new one if not found
+ * @param {string} address.toHexString()
+ * @return {*}  {Account}
```

Comment:
Is there an extra space between `}` and `{` ? Not sure on formatting conventions but wanted to just check, here and elsewhere with the similar notation.

Reply:
Hm, not 100% on that, its a VSCode prompt called Document This, and that is how it generated it.

Reply:
done in https://github.com/valorem-labs-inc/valorem-subgraph/commit/c30b7972063c3b6bec983284f757d9febe3f7a67